### PR TITLE
EFGS: Configurable extended logging

### PR DESCRIPTION
**Pull request by jendakol**
Making some logging configurable. One would use TRACE level if Go logger has such a thing.

### CODE REVIEW

File: internal/functions/efgs/utils/utils.go [hunk 1]
```diff
@@ -11,6 +11,15 @@ import (
 //Environment Environment of EFGS.
 type Environment string
 
+var EfgsExtendedLogging = false
```
Comment (dizider) [inline internal/functions/efgs/utils/utils.go:15]: Please add comment about `EfgsExtendedLogging`, otherwise linter will go crazy.

**Review by dizider — CHANGES_REQUESTED** [review]
Just one comment missing.

File: internal/functions/efgs/utils/utils.go (hunk 1 above)
Comment (jendakol) [reply internal/functions/efgs/utils/utils.go:15]: Ooops. FIxed.

**Review by dizider — APPROVED** [review]
lgtm

**Pull request merged**